kie-server-parent/kie-server-controller/kie-server-controller-client/src/main/java/org/kie/server/controller/client/rest/RestKieServerControllerClient.java

```diff
@@ -203,6 +203,11 @@ public void upgradeContainer(ContainerSpecKey containerSpecKey,
         throwUnsupportedException();
     }
 
+    @Override
+    public void deleteServerInstance(ServerInstanceKey serverInstanceKey) {
+        throwUnsupportedException();
```

Comment:
@cristianonicolai Do you plan to add support of this feature (and other currently unsupported features) for REST controller later?

Reply:
@sutaakar I'm currently focusing on the websocket protocol as the primary option to be used from the workbench when connecting to the standalone controller. Once that is done I can look into extending the REST operations.

Reply:
Ok, thanks.